### ISSUE TITLE
Mojo::IRC is failing on Win32

### ISSUE THREAD

**Issue by jhthorsen** [jhthorsen/mojo-irc#9]
http://www.cpantesters.org/cpan/report/c3d6464d-6bf7-1014-be1a-716fa945bbef

```
Output from 'C:\strawberry181\c\bin\dmake.exe test':

C:\strawberry181\perl\bin\perl.exe "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib\lib', 'blib\arch')" t/*.t
Too late to run CHECK block at C:/strawberry181/perl/site/lib/EV.pm line 123.
t/00-basic.t .............. ok
Possible attempt to put comments in qw() list at t/ctcp.t line 46.
Name "Mojo::IRC::time" used only once: possible typo at t/ctcp.t line 5.
Name "Mojo::IRC::localtime" used only once: possible typo at t/ctcp.t line 4.
# Connected
t/ctcp.t .................. ok
t/default-handlers.t ...... ok
Name "main::buf" used only once: possible typo at t/disconnect.t line 13.
Name "main::close" used only once: possible typo at t/disconnect.t line 14.
t/disconnect.t ............ ok
Mojo::Reactor::EV: Read failed: Mojo::IRC: Bad file descriptor at C:/strawberry181/perl/site/lib/Mojo/EventEmitter.pm line 17.

#   Failed test 'could still connect'
#   at t/error.t line 54.
#          got: 'close'
#     expected: 'connected'
# Looks like you failed 1 test of 4.
t/error.t ................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests 
# read irc.perl.org
# read welcome
# read join.mojo
t/join.t .................. ok
t/nick.t .................. ok
t/offline-mode-avatar.t ... ok
t/offline-mode-err.t ...... ok
t/offline-mode-numeric.t .. ok
# PASS s4cret
# NICK fooman
# USER foo 8 * :the end
t/register.t .............. ok
t/tls.t ................... skipped: Need TEST_TLS=1

Test Summary Report
-------------------
t/error.t               (Wstat: 256 Tests: 4 Failed: 1)
  Failed test:  4
  Non-zero exit status: 1
Files=12, Tests=42, 14 wallclock secs ( 0.08 usr +  0.12 sys =  0.20 CPU)
Result: FAIL
Failed 1/12 test programs. 1/42 subtests failed.
dmake.exe:  Error code 255, while making 'test_dynamic'
```

Do we care @marcusramberg ..? I don't mind closing this with "wontfix" and just skip the test on Win32 until someone complains.
